### PR TITLE
Log thread count

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

**Review by steveburton4 — APPROVED** [review]

File: src/main/kotlin/app/configuration/JobConfiguration.kt [hunk 1]
```diff
@@ -18,7 +18,9 @@ import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Profile
 import org.springframework.core.task.SimpleAsyncTaskExecutor
+import org.springframework.core.task.TaskExecutor
 import org.springframework.retry.policy.SimpleRetryPolicy
+import uk.gov.dwp.dataworks.logging.DataworksLogger
```
Comment (dan-hill2802) [inline src/main/kotlin/app/configuration/JobConfiguration.kt:23]: if this is new, can it be `DataWorksLogger`?

Comment (DanielChicot) [reply src/main/kotlin/app/configuration/JobConfiguration.kt:23]: Newish, we use it across the board,  logs everything in insight friendly json format.

Comment (DanielChicot) [reply src/main/kotlin/app/configuration/JobConfiguration.kt:23]: It's a month or so old and is in another repo.

Comment (dan-hill2802) [reply src/main/kotlin/app/configuration/JobConfiguration.kt:23]: that's fine, if it was new today then worth changing, otherwise not

**Pull request merged**